changelog.d/3679.fixed

```diff
@@ -0,0 +1 @@
+Fix Windows import
```

Comment:
I don't think that this is necessarily a fix for the windows import per-se but rather that we make ourselves compatible with older versions of the `file` binary, right?

Reply:
Yes, that's absolutely right. Supported distributions like RedHat/Rocky 8 use the older version of the file binary.

Reply:
Then let's state that in the changelog. I think this message doesn't describe the change correctly.